app/editor/src/features/content/constants/AdvancedSearchKeys.ts

```diff
@@ -13,4 +13,5 @@ export enum AdvancedSearchKeys {
   PublishedOn = 'publishedOn',
   CreatedOn = 'createdOn',
   UpdatedOn = 'updatedOn',
+  Body = 'body',
```

Comment:
Did you confirm that this will result in searching the body of the story?

Reply:
Yes I have!